geode-core/src/main/java/org/apache/geode/management/internal/configuration/converters/RegionConverter.java

```diff
@@ -75,6 +75,8 @@ protected Region fromNonNullXmlObject(RegionConfig xmlObject) {
         region.setExpirations(expirations);
       }
     }
+
+    List<RegionConfig.Index> indexes = xmlObject.getIndexes();
```

Comment:
What is the point of this addition? It looks like "indexes" is never used

Reply:
oh, added by accident. Good catch!